packages/core/src/components/tag/stories/tag.args.ts

```diff
@@ -0,0 +1,31 @@
+export const TagStoryArgs = {
+  decorators: [],
+  parameters: {
+    actions: {
+      handles: [],
+    },
+    docs: {
+      description: {
+        component:
+          'Tags are non-interactive elements used to categorize, highlight or organize items in the interface. Because they have short keywords, they help in the quick recognition of a category or element',
```

Comment:
(suggestion) as we are using ionic component, could u add link of original component?

check button example:

```
'Wrapper of Ionic Button component. Read the [Ionic documentation](https://ionicframework.com/docs/api/button) for more information about the available properties and possibilities.',
```

Reply:
@felipefialho done it https://github.com/juntossomosmais/atomium/pull/261/commits/53e84645faacde8a30a701f12705a10aada10086